ffiec/d_1/d_1_rm.sp

```diff
@@ -0,0 +1,51 @@
+benchmark "ffiec_d_1_rm" {
+  title       = "Risk Management (RM)"
+  description = "Risk Management includes a risk management program, risk assessment process, and audit function to effectively manage risk and assess the effectiveness of key controls."
+  children = [
+    benchmark.ffiec_d_1_rm_rm_b_1,
+    benchmark.ffiec_d_1_rm_ra_b_2
+  ]
+
+  tags = local.ffiec_d_1_common_tags
+}
+
+benchmark "ffiec_d_1_rm_rm_b_1" {
+  title       = "D1.RM.Rm.B.1"
```

Comment:
I think we should use the titles as per the official documentation. We could change them once - https://github.com/awsdocs/aws-config-developer-guide/issues/21 is resolved